### DISCUSSION TITLE
Github Copilot Fails

### DISCUSSION THREAD

**Discussion by Tuhin-thinks** [community/community#11344] [Copilot Conversations] [answered]
If I open GitHub copilot extension in vscode and pycharm (at separate times) for the same folder (Python based project), it stops working for that project.
For other projects (python based) it is still working fine.

**Comment by davecheney** [comment]
I'm sorry you're having difficulties. Would you be able to tell us more about what you've tried and what you saw when you tried. Screenshots or recordings would be ideal.

**Comment by Tuhin-thinks** [comment, marked as answer]
Surprisingly, after the latest update that was released 1-2hrs back, it fixed my issue, so thanks a lot for quick action!

**Reply by jansorg** [reply]
Thanks! We had a few issues with v1.1.9 of the IntelliJ plugin that completions were not showing up anymore for a few users. This is unrelated to VSCode, though. But I'm glad things are working for you again :)
